pkg/cmd/kitimport/kitimport.go

```diff
@@ -204,3 +205,25 @@ func promptToEditKitfile(contextDir, tag string, currentKitfile *artifact.KitFil
 	}
 	return readExistingKitfile(kitfilePath)
 }
+
+func getEditorName() (string, error) {
+	// Default: use the editor in the standard $EDITOR environment variable
+	editor := os.Getenv("EDITOR")
+	if editor != "" {
+		return editor, nil
```

Comment:
Would doing an "exec.LookPath()" here to check if the editor really exists makes sense?

Reply:
I don't think so -- if we couldn't find the editor in the path, we'd probably just return an error message saying that, which is what already happens:

```
Would you like to edit Kitfile before packing? (y/N): y
[ERROR] Error running external editor: exec: "nope": executable file not found in $PATH
```